cli/cmd/uninstall.go

```diff
@@ -143,6 +157,36 @@ func UninstallClusterResources(ctx context.Context, client *kube.Client, ns stri
 
 }
 
+func waitForPodsToRolloutWithoutInstrumentation(ctx context.Context, client *kube.Client) {
+	instrumentedPodReq, _ := k8slabels.NewRequirement(k8sconsts.OdigosAgentsMetaHashLabel, selection.Exists, []string{})
+	fmt.Printf("Waiting for pods to rollout without instrumentation...\n")
+
+	pollErr := wait.PollUntilContextTimeout(ctx, 10*time.Second, 5*time.Minute, true, func(innerCtx context.Context) (bool, error) {
+		pods, err := client.CoreV1().Pods("").List(innerCtx, metav1.ListOptions{
+			LabelSelector: instrumentedPodReq.String(),
+		})
+		if err != nil {
+			return false, err
+		}
+		if len(pods.Items) == 0 {
+			l := log.Print("All pods rolled out without instrumentation")
+			l.Success()
+			return true, nil
+		}
+		log.Print(fmt.Sprintf("\tWaiting for %d pods to rollout without instrumentation...\n", len(pods.Items)))
+		return false, nil
+	})
+
+	if pollErr != nil {
+		if errors.Is(pollErr, context.DeadlineExceeded) {
+			fmt.Printf("\033[33m!\tWARN\033[0m deadline exceeded for waiting pods to roll out cleanly\n")
+		}
+		if errors.Is(pollErr, context.Canceled) {
+			fmt.Printf("\033[33m!\tWARN\033[0m context canceled for waiting pods to roll out cleanly\n")
```

Comment:
"context canceled" is an implementation detail of golang, and not a message we should show users of CLI in my opinion